app/Providers/EventServiceProvider.php

```diff
@@ -110,6 +110,9 @@ class EventServiceProvider extends ServiceProvider
         DiscordUnlinked::class => [
             RemoveDiscordUser::class,
         ],
+        \App\Events\Mship\Endorsement\TierEndorsementAdded::class => [
+            \App\Listeners\Mship\Endorsement\NotifyOfTierEndorsement::class,
+        ],
```

Comment:
Import rather than fully qualified

Reply:
General convention in that file seems to be full path if you look at 95% of the mappings. Happy to go either way but the file would be 95% imports

Reply:
One of these days I'll just import everything across the project! Fully qualified looks messy